<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Ekran" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="wyjdz">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QSlider" name="x_slider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-160</number>
    </property>
    <property name="maximum">
     <number>160</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QSlider" name="y_slider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-160</number>
    </property>
    <property name="maximum">
     <number>160</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="z_slider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-160</number>
    </property>
    <property name="maximum">
     <number>160</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rotacja_z">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rotacja_x">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>303</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rotacja_y">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="skalowanie_y">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>130</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="skalowanie_x">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>196</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>130</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="skalowanie_z">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>130</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="widoczne">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>tylko widoczne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tekstura">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>430</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>tekstura</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ekran</class>
   <extends>QWidget</extends>
   <header>ekran.h</header>
   <container>1</container>
   <slots>
    <signal>setCurrentAlpha(int)</signal>
    <slot>scaleX(int)</slot>
    <slot>scaleY(int)</slot>
    <slot>scaleZ(int)</slot>
    <slot>rotX(int)</slot>
    <slot>rotY(int)</slot>
    <slot>rotZ(int)</slot>
    <slot>transX(int)</slot>
    <slot>transY(int)</slot>
    <slot>transZ(int)</slot>
    <slot>przesX(int)</slot>
    <slot>przesY(int)</slot>
    <slot>skalX(int)</slot>
    <slot>skalY(int)</slot>
    <slot>rotacja(int)</slot>
    <slot>pochX(int)</slot>
    <slot>pochY(int)</slot>
    <slot>ogniskowa(int)</slot>
    <slot>stateChanged(int)</slot>
    <slot>teksturaChanged(int)</slot>
    <slot>alphaSliderChange(int)</slot>
    <slot>warstwaButton()</slot>
    <slot>alphaSliderChange()</slot>
    <slot>setAlpha(int)</slot>
    <slot>blendMode()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
